Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/util/block/GenChunk.java

```diff
@@ -109,7 +110,7 @@ public boolean setBiome(int x, int z, Biome biome) {
     }
 
     @Override public boolean setBlock(int x, int y, int z, @NotNull Pattern pattern) {
-        return setBlock(x, y, z, PatternUtil.apply(pattern, x, y, z));
+        return setBlock(x, y, z, PatternUtil.apply(Preconditions.checkNotNull(pattern, "Pattern may not be null"), x, y, z));
```

Comment:
It shouldn't be null anyways because of the annotation we have on this method.

Reply:
It was null when the patterns failed to parse and it was really hard to track down.